pypeit/processimages.py

```diff
@@ -400,10 +400,9 @@ def flat_field(self, pixel_flat, bpm, slitprofile=None):
 
         # Flat-field the data and return the result
         self.stack = flat.flatfield(self.stack, self.pixel_flat, self.bpm,
-                                      slitprofile=self.slitprof)
+                                    slitprofile=self.slitprof)
```

Comment:
Since we always compute a pixel flat and a slit profile, I would probably slightly prefer that slitprofile be a required rather than an optional argument here.

Reply:
Done.